### DISCUSSION TITLE
ESPAltherma turned 2!

### DISCUSSION THREAD

**Discussion by raomin** [raomin/ESPAltherma#200] [Announcements]
Two years ago I pushed the first commit on this repository. What started as a personal project quickly became a community adventure with hundreds of adoptions across the world, shared experiences and active discussions on the improvement of ESPAltherma.

It feels so great to see people happy and proud of their installations. What I like most are the comments like "it's the first time I hack on of my device and it worked!". That's the exact reason why I did this, give to owners of Altherma heat pump, the possibility to finally get critical information for anyone who is interested in the home automation.

Looking back at this two years, I'm also amazed how ESPAltherma evolved. Sure the core functions and data didn't change much, but ESPAltherma is now more stable and we are now supporting protocol S to cover even more heat pumps.

All this is thanks to the community activity but also to the amazing supporters 💗. All this keeps me motivated on taking ESPAltherma even further.

But that's what I have in mind for future evolutions: 
_Of course, the roadmap is to be built together as it was always the case. So feel free to add comments and suggestions!_
 - Support of power level limits (see [discussion here](https://github.com/raomin/ESPAltherma/discussions/170)) 
 - Improved logs
 - A clear way for people to choose the appropriate definition file > Documentation or even better: autodiscovery (we check couple of registries to identify the model)
 - A web interface for configuration and log history (this would open the possibility for easy flashing with webserial, configured devices ready to be shipped, etc...)
 - I'm still hopeful that one day we'll manage to get more information on the configuration capabilities of the protocol. Who knows, maybe Daikin will see some interest in this...!
 

Thanks for these last 2 years, can't wait to see how far we can go together!

**Comment by alexdelli** [comment]
Oh Romin thank you! without your work I could not test my heating system to find the best settings to save money.

I'm excited for the power limit levels, because my HP is oversized, so to avoid frequently on/off I need a system to limit power different than silente mode. I wish this hack can help me.

Thank you again!!

(in an italian facebook group we are a lot of people to use Espaltherma :)  )

**Reply by GpMissi** [reply]
Ciao Alex, di che gruppo si tratta? Potresti mandarmi un link? Grazie :-)

**Reply by alexdelli** [reply]
it's a thread in a group POMPE DI CALORE PER TUTTI

**Reply by xalex75** [reply]
Many thanks to you raomin!

@alexdelli I'll subscribe to that group immediately! ;-)
(mmmm Pompe di calore per tutti o Pompe di calore per tutti - Persone serie?)

**Reply by stefo8** [reply]
> (mmmm Pompe di calore per tutti o Pompe di calore per tutti - Persone serie?)

Ciao, penso il primo mi puoi confermare? Grazie

**Reply by alexdelli** [reply]
Si si, il primo
